docs/source/advanced/boosted_frame.rst

```diff
@@ -11,7 +11,7 @@ how to handle the most important aspects of boosted-frame simulations with FBPIC
     - Converting the input parameters from the lab frame to the boosted frame
     - Converting simulation results from the boosted frame to the lab frame
     - Avoiding the numerical Cherenkov instability (NCI)
-    - Estimating the speedup (number of iterations) in the boosted frame
+    - Estimating the number of iterations to use in the boosted frame
```

Comment:
As you mentioned, estimating the actual speed-up is tricky.
And from the strict point of view of managing to run their first boosted-frame simulation, I guess a user cares more about the practical aspect, i.e. which number of iterations to use in their script.

Reply:
:white_check_mark: